R/clustering.R

```diff
@@ -548,12 +548,9 @@
 #' the main effects.
 #'
 #' If you have column effects alongside row clusters (they are not permitted
-#' alongside column clusters), without interactions, i.e. the formula
-#'
-#' \code{Y ~ ROWCLUST + COL}
-#'
-#' with Binary model Logit(P(Y = 1)) = mu + rowc_coef_r + col_coef_j
-#'
+#' alongside column clusters), without interactions, i.e. the formula 
```

Comment:
I disagree with this change because I think it makes more sense to keep the Logit equation on a single separate line